frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -605,8 +619,77 @@ function ReaderFooter:init()
     end
 
     self.visibility_change = nil
+
+    self.custom_text = G_reader_settings:readSetting("reader_footer_custom_text") or "KOReader"
+    self.custom_text_repetitions =
+        tonumber(G_reader_settings:readSetting("reader_footer_custom_text_repetitions")) or 1
```

Comment:
There was a recommendation of using a default value: https://github.com/koreader/koreader/pull/7371.
https://github.com/koreader/koreader/pull/7371#issuecomment-788157906

Reply:
Thank you, that one I have missed.